.travis.yml

```diff
@@ -0,0 +1,5 @@
+---
+language: ruby
+cache: bundler
+rvm:
+- 2.5.6
```

Comment:
Preferably I would like to test more than a single version of ruby.  Again, since I wanted this to be a script, this was designed to work on ruby even as far back as 1.8.7.  Not that we use that, but that was the idea for not making use of conveniences like `<<~` for HEREDOC strings.

Reply:
I have to use *something* for travis tests...we can always add more.